sdk/cloudmachine/Azure.Projects/src/Azure.Projects.csproj

```diff
@@ -12,6 +12,7 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <PackageReference Include="Azure.AI.Agents" VersionOverride="1.0.0-alpha.20250211.4" />
```

Comment:
Why are you directly depending on an alpha version here?

Reply:
This library has never GA, so I have to depend on alpha.

Reply:
I removed that. The same APIs exist in Azure.AI.Projects.